SU-SOE/engineering_profile
# SOED9-77: added engineering_profile_helper and vocabularies

# READY FOR REVIEW

# Summary
- In this PR, we add the basic module files for `engineering_profile_helper`. We add two new vocabularies, as well as default content for the taxonomy_terms in those vocabs. We also add the `Departments` vocabulary to Basic Pages.  Tests for the default content is included as well.

# Needed By (Date)
- ASAP

# Urgency
- Medium

# Steps to Test

1. Checkout this branch.
2. do a `blt di` to reinstall your site.
3. Check to make sure `Department` and `Magazine Topics` vocabs exist and have terms in them.
4. Make a basic page with a Department selected.
5. Verify the tests pass on circleci

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
